docs/user_guide/branding.rst

```diff
@@ -74,6 +74,22 @@ To reference an external website, make sure your link starts with ``http``:
        }
    }
 
+Customize logo alternative text
+-------------------------------
+
+The logo is by default using "Logo image" as an alternative text of the image. To set
+a specific alt text for this to be accessible to those using screen readers or other
+assistive tech, use the ``html_teme_options["logo"]["alt_text"]`` and provide a new name
+as in the following example:
+
+.. code-block:: python
+
+   html_theme_options = {
+       "logo": {
+           "alt_text": "foo",
+       }
+   }
```

Comment:
```suggestion
.. code-block:: python
   :caption: conf.py

   html_theme_options = {
       "logo": {
           "alt_text": "foo",
       }
   }
```

What do you think about starting to use `:caption:` in order to make it clear where the config exists?

Reply:
I didn't know captions where an option for the code block directive, I really like it